### PR TITLE
Poc demo

### PR DESCRIPTION
**Pull request by mngmay**
Banner added and styles

### CODE REVIEW

File: components/cards/EventWelcomeCard.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+const EventWelcomeCard = ({ event }) => {
+  //TODO: replace with dynamic props
+  return (
+    <div className={cx("event-card")}>
+      <p className={cx("title")}>Welcome to IAC22</p>
+      <p className={cx("description")}>
+        Some type of specific introduction and explanation on what the following
+        events are for. Example; Here you will find events on blank, they are
+        intended to do blank. something about social connections.
+      </p>
```
Comment (fastestOrange) [inline components/cards/EventWelcomeCard.jsx:17]: You would be one step closer towards implementing the todo above defining a mock props object and using that to drive the placeholder UI.

Comment (fastestOrange) [reply components/cards/EventWelcomeCard.jsx:17]: instead of hardcoding the text into the markup

**Review by fastestOrange — APPROVED** [review]
Great! Small suggestion for future work

File: components/cards/EventWelcomeCard.jsx (hunk 1 above)
Comment (mngmay) [reply components/cards/EventWelcomeCard.jsx:17]: Totally agree! I'll be replacing this with a placeholder and apply that approach for future PRs.

**Pull request merged**